package.json

```diff
@@ -118,6 +119,7 @@
     "ngx-md": "^7.0.0",
     "ngx-pagination": "^3.2.1",
     "node-http-proxy-json": "^0.1.9",
+    "npm": "^6.14.11",
```

Comment:
is this npm necessary looks like it refresh the whole yarn.lock file?

Reply:
This must have been added when I was trying to resolve an issue with the rpx-xui-common-lib. I have removed it from the file now.